source/instance.txt

```diff
@@ -23,18 +32,22 @@ MongoDB instance to which Compass is connected, including:
 - :doc:`Performance statistics </performance>`, including memory
```

Comment:
two things: 

1. now that you can see more than one instance, we should make this more general to talk about listing connections & not databases 
2. previously the Performance Tab was it's own tab on the homepage. Now this (and the databases tab actually) are accessed through the connections sidebar. It feels a little weird to discuss these as aspects of the home page, maybe we could just link to the "Connections Sidebar" page since that goes over all of the actions. What do you think?

Reply:
I moved info from the Databases and Performance sections on the Home page to the Connections Sidebar page and removed those sections from the Home page. What do you think?

Reply:
This makes sense to me. I find it a little odd that the "Performance" tab still falls under "Compass Home," do you think we should pull it out?

Reply:
I moved it under "Interact with your Connections" in [DOCSP-42696](https://jira.mongodb.org/browse/DOCSP-42696)!